app/assets/stylesheets/views/_answers.scss

```diff
@@ -0,0 +1,9 @@
+.answers {
+  .add-title-margin {
+    @include responsive-top-margin;
+  }
+
+  strong {
```

Comment:
i dont really know why tbh, as i don't have the history for that, some formats have it some don't